recipes/xlsxio/all/conanfile.py

```diff
@@ -47,19 +47,22 @@ def configure(self):
         self.settings.rm_safe("compiler.libcxx")
         self.settings.rm_safe("compiler.cppstd")
 
+        if self.options.with_wide:
+            self.options["expat"].char_type = "ushort"
+        if self.options.get_safe("with_minizip_ng"):
+            self.options["minizip-ng"].mz_compatibility = True
```

Comment:
Can I get some logs that show that this compiles succesfully/works as expected? CCI won't get to check these configs, so it would be nice to have a report on whether that builds as of now :)

Reply:
@RubenRBS 
Thank you for your review!
As you know, there is an issue about minizip-ng.

```
ERROR: Provide Conflict: Both 'minizip-ng/3.0.8' and 'minizip-ng/3.0.8' provide 'minizip'.
```

I have already created another PR to solve it: https://github.com/conan-io/conan-center-index/pull/18465.

Reply:
Oops, didn't see that one, thanks for the heads up :)

Reply:
@RubenRBS 
Thank you for your review!
PR for minizip-ng was alread merged.

Here is a log for `with_minizip_ng=True`:

```
> conan remove -c xlsxio; conan create .  -o xlsxio/\*:with_minizip_ng=True  --build=missing --version=0.2.34
Remove summary:
Local Cache
  xlsxio/0.2.34#cd3fd649b59a7a84190e9ad500acabb5: Removed recipe and all binaries

======== Exporting recipe to the cache ========
xlsxio/0.2.34: Exporting package recipe: /home/toge/src/conan-center-index/recipes/xlsxio/all/conanfile.py
xlsxio/0.2.34: exports: File 'conandata.yml' found. Exporting it...
xlsxio/0.2.34: Calling export_sources()
xlsxio/0.2.34: Copied 1 '.yml' file: conandata.yml
xlsxio/0.2.34: Copied 1 '.py' file: conanfile.py
xlsxio/0.2.34: Copied 1 '.patch' file: 0234_cmake.patch
xlsxio/0.2.34: Exported to cache folder: /home/toge/.conan2/p/xlsxi5ea4f45fc9890/e
xlsxio/0.2.34: Exported: xlsxio/0.2.34#cd3fd649b59a7a84190e9ad500acabb5 (2023-07-17 16:07:09 UTC)

======== Input profiles ========
Profile host:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=20
compiler.libcxx=libstdc++11
compiler.version=13
os=Linux
[options]
xlsxio/*:with_minizip_ng=True

Profile build:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=20
compiler.libcxx=libstdc++11
compiler.version=13
os=Linux


======== Computing dependency graph ========
Graph root
    cli
Requirements
    bzip2/1.0.8#411fc05e80d47a89045edc1ee6f23c1d - Cache
    expat/2.5.0#7fde619c77c77a50ca72364bc92c4907 - Cache
    libiconv/1.17#fa54397801cd96911a8294bc5fc76335 - Cache
    minizip-ng/3.0.8#15d79b52a8441abb6839d4ad98407685 - Cache
    openssl/3.1.1#3a25e05b364f335633143656dc265841 - Cache
    xlsxio/0.2.34#cd3fd649b59a7a84190e9ad500acabb5 - Cache
    xz_utils/5.4.2#b6ee8320403def553418874435445982 - Cache
    zlib/1.2.13#e377bee636333ae348d51ca90874e353 - Cache
    zstd/1.5.5#93372fe14bb7883bd4de82914e0a1841 - Cache
Build requirements
    meson/1.0.0#15586c0ac6f682805875ef903dbe7ee2 - Cache
    ninja/1.11.1#77587f8c8318662ac8e5a7867eb4be21 - Cache
    pkgconf/1.9.5#743ca0d41d35a84b1f89af337ddaa1a0 - Cache
Resolved version ranges
    openssl/[>=1.1 <4]: openssl/3.1.1

======== Computing necessary packages ========
Requirements
    bzip2/1.0.8#411fc05e80d47a89045edc1ee6f23c1d:432d7695adc526147073285e155d2ca0340bc76d#2a24b1ce3cdda308e83c1e139f22de32 - Cache
    expat/2.5.0#7fde619c77c77a50ca72364bc92c4907:881eecfe39e91e6c39fcf5547043d4ae662b1541#c11f50ac22411a0d9e67bf17060245cb - Cache
    libiconv/1.17#fa54397801cd96911a8294bc5fc76335:c81087b06d1a5beef0ad711fc32d45f7a425a8f3#4afe9416cf9115775f83e83611854c18 - Cache
    minizip-ng/3.0.8#15d79b52a8441abb6839d4ad98407685:3b5259ef3cc6329607ff551fde193a41583f8ccf#ec3e303e40716979ed6cd6a5ef5ddb7e - Cache
    openssl/3.1.1#3a25e05b364f335633143656dc265841:096c83f1c40759aa21cef99874d8e95faa1ec319#1b53d8e003fa644debb2dc19948a8e73 - Cache
    xlsxio/0.2.34#cd3fd649b59a7a84190e9ad500acabb5:ebece53a55a63805917db75f2261416b355ed6ca - Build
    xz_utils/5.4.2#b6ee8320403def553418874435445982:c81087b06d1a5beef0ad711fc32d45f7a425a8f3#cf4d822c688271c5525343bc07da722f - Cache
    zlib/1.2.13#e377bee636333ae348d51ca90874e353:c81087b06d1a5beef0ad711fc32d45f7a425a8f3#be19f485811d66a68a3c9848cc095a46 - Cache
    zstd/1.5.5#93372fe14bb7883bd4de82914e0a1841:ef043a2a516d05649c001063aa1b140e18f115d8#918372c071deedc79fefb46c64715c31 - Cache
Build requirements
Skipped binaries
    meson/1.0.0, ninja/1.11.1, pkgconf/1.9.5

======== Installing packages ========
bzip2/1.0.8: Already installed! (1 of 9)
expat/2.5.0: Already installed! (2 of 9)
libiconv/1.17: Already installed! (3 of 9)
xz_utils/5.4.2: Already installed! (4 of 9)
zlib/1.2.13: Already installed! (5 of 9)
zstd/1.5.5: Already installed! (6 of 9)
openssl/3.1.1: Already installed! (7 of 9)
minizip-ng/3.0.8: Already installed! (8 of 9)
xlsxio/0.2.34: Calling source() in /home/toge/.conan2/p/xlsxi5ea4f45fc9890/s/src

-------- Installing package xlsxio/0.2.34 (9 of 9) --------
xlsxio/0.2.34: Building from source
xlsxio/0.2.34: Package xlsxio/0.2.34:ebece53a55a63805917db75f2261416b355ed6ca
xlsxio/0.2.34: Copying sources to build folder
xlsxio/0.2.34: Building your package in /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/b
xlsxio/0.2.34: Calling generate()
xlsxio/0.2.34: Generators folder: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/b/build/Release/generators
xlsxio/0.2.34: CMakeToolchain generated: conan_toolchain.cmake
xlsxio/0.2.34: CMakeToolchain generated: CMakePresets.json
xlsxio/0.2.34: CMakeToolchain generated: ../../../src/CMakeUserPresets.json
xlsxio/0.2.34: Generating aggregated env files
xlsxio/0.2.34: Generated aggregated env files: ['conanbuild.sh', 'conanrun.sh']
xlsxio/0.2.34: Calling build()
xlsxio/0.2.34: Apply patch (conan): conan build cmake support
xlsxio/0.2.34: Running CMake.configure()
xlsxio/0.2.34: RUN: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/b/build/Release/generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/b/src"
CMake Deprecation Warning at CMakeLists.txt:1 (CMAKE_MINIMUM_REQUIRED):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Using Conan toolchain: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/b/build/Release/generators/conan_toolchain.cmake
-- Conan toolchain: Setting CMAKE_POSITION_INDEPENDENT_CODE=ON (options.fPIC)
-- Conan toolchain: Setting BUILD_SHARED_LIBS = OFF
-- The C compiler identification is GNU 13.1.1
-- The CXX compiler identification is GNU 13.1.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- XLSX I/O library version: 0.2.34
-- Conan: Component target declared 'MINIZIP::minizip'
-- Conan: Target declared 'BZip2::BZip2'
-- Conan: Including build module from '/home/toge/.conan2/p/bzip226aba6bc6c545/p/lib/cmake/conan-official-bzip2-variables.cmake'
-- Conan: Target declared 'LibLZMA::LibLZMA'
-- Conan: Including build module from '/home/toge/.conan2/p/xz_ut4feb041f996af/p/lib/cmake/conan-official-xz_utils-variables.cmake'
-- Conan: Component target declared 'zstd::libzstd_static'
-- Conan: Component target declared 'OpenSSL::Crypto'
-- Conan: Component target declared 'OpenSSL::SSL'
-- Conan: Target declared 'openssl::openssl'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Including build module from '/home/toge/.conan2/p/b/opense61599ddda9b5/p/lib/cmake/conan-official-openssl-variables.cmake'
-- Conan: Target declared 'Iconv::Iconv'
-- Conan: Target declared 'expat::expat'
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
-- Configuring done (0.2s)
-- Generating done (0.0s)
-- Build files have been written to: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/b/build/Release

xlsxio/0.2.34: Running CMake.build()
xlsxio/0.2.34: RUN: cmake --build "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/b/build/Release" -- -j12
[ 17%] Building C object CMakeFiles/xlsxio_read_STATIC.dir/lib/xlsxio_read_sharedstrings.c.o
[ 17%] Building C object CMakeFiles/xlsxio_read_STATIC.dir/lib/xlsxio_read.c.o
[ 17%] Building C object CMakeFiles/xlsxio_write_STATIC.dir/lib/xlsxio_write.c.o
[ 23%] Linking C static library libxlsxio_write.a
[ 23%] Built target xlsxio_write_STATIC
[ 41%] Building C object CMakeFiles/example_xlsxio_write.dir/examples/example_xlsxio_write.c.o
[ 41%] Building C object CMakeFiles/xlsxio_csv2xlsx.dir/src/xlsxio_csv2xlsx.c.o
[ 41%] Building C object CMakeFiles/example_xlsxio_write_getversion.dir/examples/example_xlsxio_write_getversion.c.o
[ 47%] Linking C executable example_xlsxio_write_getversion
[ 52%] Linking C executable example_xlsxio_write
[ 58%] Linking C executable xlsxio_csv2xlsx
[ 64%] Linking C static library libxlsxio_read.a
[ 64%] Built target xlsxio_read_STATIC
[ 82%] Building C object CMakeFiles/example_xlsxio_read.dir/examples/example_xlsxio_read.c.o
[ 82%] Building C object CMakeFiles/xlsxio_xlsx2csv.dir/src/xlsxio_xlsx2csv.c.o
[ 82%] Building C object CMakeFiles/example_xlsxio_read_advanced.dir/examples/example_xlsxio_read_advanced.c.o
[ 94%] Linking C executable example_xlsxio_read_advanced
[ 94%] Linking C executable example_xlsxio_read
[100%] Linking C executable xlsxio_xlsx2csv
[100%] Built target example_xlsxio_write_getversion
[100%] Built target example_xlsxio_write
[100%] Built target xlsxio_csv2xlsx
[100%] Built target example_xlsxio_read
[100%] Built target xlsxio_xlsx2csv
[100%] Built target example_xlsxio_read_advanced

xlsxio/0.2.34: Package 'ebece53a55a63805917db75f2261416b355ed6ca' built
xlsxio/0.2.34: Build folder /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/b/build/Release
xlsxio/0.2.34: Generating the package
xlsxio/0.2.34: Temporary package folder /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p
xlsxio/0.2.34: Calling package()
xlsxio/0.2.34: Running CMake.install()
xlsxio/0.2.34: RUN: cmake --install "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/b/build/Release" --prefix "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p"
-- Install configuration: "Release"
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/lib/libxlsxio_read.a
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/lib/libxlsxio_write.a
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/cmake/xlsxioTargets.cmake
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/cmake/xlsxioTargets-release.cmake
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/cmake/xlsxio-config.cmake
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/xlsxio_xlsx2csv
-- Set runtime path of "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/xlsxio_xlsx2csv" to ""
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/xlsxio_csv2xlsx
-- Set runtime path of "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/xlsxio_csv2xlsx" to ""
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/example_xlsxio_write_getversion
-- Set runtime path of "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/example_xlsxio_write_getversion" to ""
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/example_xlsxio_write
-- Set runtime path of "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/example_xlsxio_write" to ""
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/example_xlsxio_read
-- Set runtime path of "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/example_xlsxio_read" to ""
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/example_xlsxio_read_advanced
-- Set runtime path of "/home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/bin/example_xlsxio_read_advanced" to ""
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/include
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/include/xlsxio_read.h
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/include/xlsxio_version.h
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/include/xlsxio_write.h
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/cmake/libxlsxio_writeConfig.cmake
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/cmake/libxlsxio_writeConfigVersion.cmake
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/lib/pkgconfig/libxlsxio_write.pc
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/cmake/libxlsxio_readConfig.cmake
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/cmake/libxlsxio_readConfigVersion.cmake
-- Installing: /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p/lib/pkgconfig/libxlsxio_read.pc

xlsxio/0.2.34: package(): Packaged 1 '.txt' file: LICENSE.txt
xlsxio/0.2.34: package(): Packaged 2 '.a' files: libxlsxio_read.a, libxlsxio_write.a
xlsxio/0.2.34: package(): Packaged 6 files
xlsxio/0.2.34: package(): Packaged 3 '.h' files: xlsxio_read.h, xlsxio_version.h, xlsxio_write.h
xlsxio/0.2.34: Created package revision 07a722e67ad415c3f6f29df76c1ea6b8
xlsxio/0.2.34: Package 'ebece53a55a63805917db75f2261416b355ed6ca' created
xlsxio/0.2.34: Full package reference: xlsxio/0.2.34#cd3fd649b59a7a84190e9ad500acabb5:ebece53a55a63805917db75f2261416b355ed6ca#07a722e67ad415c3f6f29df76c1ea6b8
xlsxio/0.2.34: Package folder /home/toge/.conan2/p/b/xlsxi1d4b8f74bee29/p
WARN: deprecated: Usage of deprecated Conan 1.X features that will be removed in Conan 2.X:
WARN: deprecated:     'cpp_info.names' used in: libiconv/1.17, zstd/1.5.5, openssl/3.1.1, bzip2/1.0.8, xz_utils/5.4.2, expat/2.5.0, zlib/1.2.13, minizip-ng/3.0.8, xlsxio/0.2.34
WARN: deprecated:     'cpp_info.build_modules' used in: xz_utils/5.4.2, openssl/3.1.1, bzip2/1.0.8
WARN: deprecated:     'env_info' used in: libiconv/1.17, openssl/3.1.1, bzip2/1.0.8
WARN: deprecated:     'cpp_info.filenames' used in: minizip-ng/3.0.8

======== Launching test_package ========

======== Computing dependency graph ========
Graph root
    xlsxio/0.2.34 (test package): /home/toge/src/conan-center-index/recipes/xlsxio/all/test_package/conanfile.py
Requirements
    bzip2/1.0.8#411fc05e80d47a89045edc1ee6f23c1d - Cache
    expat/2.5.0#7fde619c77c77a50ca72364bc92c4907 - Cache
    libiconv/1.17#fa54397801cd96911a8294bc5fc76335 - Cache
    minizip-ng/3.0.8#15d79b52a8441abb6839d4ad98407685 - Cache
    openssl/3.1.1#3a25e05b364f335633143656dc265841 - Cache
    xlsxio/0.2.34#cd3fd649b59a7a84190e9ad500acabb5 - Cache
    xz_utils/5.4.2#b6ee8320403def553418874435445982 - Cache
    zlib/1.2.13#e377bee636333ae348d51ca90874e353 - Cache
    zstd/1.5.5#93372fe14bb7883bd4de82914e0a1841 - Cache
Build requirements
    meson/1.0.0#15586c0ac6f682805875ef903dbe7ee2 - Cache
    ninja/1.11.1#77587f8c8318662ac8e5a7867eb4be21 - Cache
    pkgconf/1.9.5#743ca0d41d35a84b1f89af337ddaa1a0 - Cache

======== Computing necessary packages ========
Requirements
    bzip2/1.0.8#411fc05e80d47a89045edc1ee6f23c1d:432d7695adc526147073285e155d2ca0340bc76d#2a24b1ce3cdda308e83c1e139f22de32 - Cache
    expat/2.5.0#7fde619c77c77a50ca72364bc92c4907:881eecfe39e91e6c39fcf5547043d4ae662b1541#c11f50ac22411a0d9e67bf17060245cb - Cache
    libiconv/1.17#fa54397801cd96911a8294bc5fc76335:c81087b06d1a5beef0ad711fc32d45f7a425a8f3#4afe9416cf9115775f83e83611854c18 - Cache
    minizip-ng/3.0.8#15d79b52a8441abb6839d4ad98407685:3b5259ef3cc6329607ff551fde193a41583f8ccf#ec3e303e40716979ed6cd6a5ef5ddb7e - Cache
    openssl/3.1.1#3a25e05b364f335633143656dc265841:096c83f1c40759aa21cef99874d8e95faa1ec319#1b53d8e003fa644debb2dc19948a8e73 - Cache
    xlsxio/0.2.34#cd3fd649b59a7a84190e9ad500acabb5:ebece53a55a63805917db75f2261416b355ed6ca#07a722e67ad415c3f6f29df76c1ea6b8 - Cache
    xz_utils/5.4.2#b6ee8320403def553418874435445982:c81087b06d1a5beef0ad711fc32d45f7a425a8f3#cf4d822c688271c5525343bc07da722f - Cache
    zlib/1.2.13#e377bee636333ae348d51ca90874e353:c81087b06d1a5beef0ad711fc32d45f7a425a8f3#be19f485811d66a68a3c9848cc095a46 - Cache
    zstd/1.5.5#93372fe14bb7883bd4de82914e0a1841:ef043a2a516d05649c001063aa1b140e18f115d8#918372c071deedc79fefb46c64715c31 - Cache
Build requirements
Skipped binaries
    meson/1.0.0, ninja/1.11.1, pkgconf/1.9.5

======== Installing packages ========
bzip2/1.0.8: Already installed! (1 of 9)
expat/2.5.0: Already installed! (2 of 9)
libiconv/1.17: Already installed! (3 of 9)
xz_utils/5.4.2: Already installed! (4 of 9)
zlib/1.2.13: Already installed! (5 of 9)
zstd/1.5.5: Already installed! (6 of 9)
openssl/3.1.1: Already installed! (7 of 9)
minizip-ng/3.0.8: Already installed! (8 of 9)
xlsxio/0.2.34: Already installed! (9 of 9)
WARN: deprecated: Usage of deprecated Conan 1.X features that will be removed in Conan 2.X:
WARN: deprecated:     'cpp_info.names' used in: libiconv/1.17, zstd/1.5.5, openssl/3.1.1, bzip2/1.0.8, xz_utils/5.4.2, expat/2.5.0, zlib/1.2.13, minizip-ng/3.0.8, xlsxio/0.2.34
WARN: deprecated:     'cpp_info.build_modules' used in: xz_utils/5.4.2, openssl/3.1.1, bzip2/1.0.8
WARN: deprecated:     'env_info' used in: libiconv/1.17, openssl/3.1.1, bzip2/1.0.8
WARN: deprecated:     'cpp_info.filenames' used in: minizip-ng/3.0.8

======== Testing the package ========
Removing previously existing 'test_package' build folder: /home/toge/src/conan-center-index/recipes/xlsxio/all/test_package/build/gcc-13-x86_64-20-release
xlsxio/0.2.34 (test package): Test package build: build/gcc-13-x86_64-20-release
xlsxio/0.2.34 (test package): Test package build folder: /home/toge/src/conan-center-index/recipes/xlsxio/all/test_package/build/gcc-13-x86_64-20-release
xlsxio/0.2.34 (test package): Writing generators to /home/toge/src/conan-center-index/recipes/xlsxio/all/test_package/build/gcc-13-x86_64-20-release/generators
xlsxio/0.2.34 (test package): Generator 'CMakeToolchain' calling 'generate()'
xlsxio/0.2.34 (test package): CMakeToolchain generated: conan_toolchain.cmake
xlsxio/0.2.34 (test package): CMakeToolchain generated: CMakePresets.json
xlsxio/0.2.34 (test package): CMakeToolchain generated: ../../../CMakeUserPresets.json
xlsxio/0.2.34 (test package): Generator 'VirtualRunEnv' calling 'generate()'
xlsxio/0.2.34 (test package): Generator 'CMakeDeps' calling 'generate()'
xlsxio/0.2.34 (test package): Generating aggregated env files
xlsxio/0.2.34 (test package): Generated aggregated env files: ['conanrun.sh', 'conanbuild.sh']

======== Testing the package: Building ========
xlsxio/0.2.34 (test package): Calling build()
xlsxio/0.2.34 (test package): Running CMake.configure()
xlsxio/0.2.34 (test package): RUN: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="/home/toge/src/conan-center-index/recipes/xlsxio/all/test_package/build/gcc-13-x86_64-20-release/generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/home/toge/src/conan-center-index/recipes/xlsxio/all/test_package" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/toge/src/conan-center-index/recipes/xlsxio/all/test_package"
-- Using Conan toolchain: /home/toge/src/conan-center-index/recipes/xlsxio/all/test_package/build/gcc-13-x86_64-20-release/generators/conan_toolchain.cmake
-- Conan toolchain: C++ Standard 20 with extensions OFF
-- The C compiler identification is GNU 13.1.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Conan: Component target declared 'xlsxio::xlsxio_read'
-- Conan: Component target declared 'xlsxio::xlsxio_write'
-- Conan: Target declared 'xlsxio::xlsxio'
-- Conan: Component target declared 'MINIZIP::minizip'
-- Conan: Target declared 'BZip2::BZip2'
-- Conan: Including build module from '/home/toge/.conan2/p/bzip226aba6bc6c545/p/lib/cmake/conan-official-bzip2-variables.cmake'
-- Conan: Target declared 'LibLZMA::LibLZMA'
-- Conan: Including build module from '/home/toge/.conan2/p/xz_ut4feb041f996af/p/lib/cmake/conan-official-xz_utils-variables.cmake'
-- Conan: Component target declared 'zstd::libzstd_static'
-- Conan: Component target declared 'OpenSSL::Crypto'
-- Conan: Component target declared 'OpenSSL::SSL'
-- Conan: Target declared 'openssl::openssl'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Including build module from '/home/toge/.conan2/p/b/opense61599ddda9b5/p/lib/cmake/conan-official-openssl-variables.cmake'
-- Conan: Target declared 'Iconv::Iconv'
-- Conan: Target declared 'expat::expat'
-- Configuring done (0.1s)
-- Generating done (0.0s)
-- Build files have been written to: /home/toge/src/conan-center-index/recipes/xlsxio/all/test_package/build/gcc-13-x86_64-20-release

xlsxio/0.2.34 (test package): Running CMake.build()
xlsxio/0.2.34 (test package): RUN: cmake --build "/home/toge/src/conan-center-index/recipes/xlsxio/all/test_package/build/gcc-13-x86_64-20-release" -- -j12
[ 50%] Building C object CMakeFiles/test_package.dir/test_package.c.o
[100%] Linking C executable test_package
[100%] Built target test_package


======== Testing the package: Executing test ========
xlsxio/0.2.34 (test package): Running test()
xlsxio/0.2.34 (test package): RUN: ./test_package
```